homeassistant/components/harmony/__init__.py

```diff
@@ -67,7 +74,7 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
 def _async_import_options_from_data_if_missing(hass: HomeAssistant, entry: ConfigEntry):
     options = dict(entry.options)
     modified = 0
-    for importable_option in [ATTR_ACTIVITY, ATTR_DELAY_SECS]:
+    for importable_option in [ATTR_ACTIVITY, ATTR_DELAY_SECS, ATTR_ACTIVITY_NOTIFY]:
```

Comment:
Since we don't update legacy platform schema, this can be removed here since it will never be imported

Reply:
Same here, removed the item I added but figuring leaving rest in there for folks coming from older HA's?

Reply:
> Same here, removed the item I added but figuring leaving rest in there for folks coming from older HA's?

Exactly